### DISCUSSION TITLE
SEO on Automad V2

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#36] [Q&A] [answered]
hi
I can assume that SEO in version 2 is handled the same as in version 1?
This means that the statements from this thread can also be transferred to V2? --> https://discuss.automad.org/d/214-seo-question

**Comment by marcantondahmen** [comment, marked as answer]
Version 2 brings some improvements to SEO. Features of version 1 will continue to exist in version 2.

**Comment by cracksilver** [comment]
thx
